test/integration/targets/nios_smoke_tests/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+- debug: msg="running"
+- include: prepare.yml
+- include: nios_dns_view.yml
+- include: nios_host_records.yml
+- include: nios_networks.yml
+- include: nios_network_views.yml
+- include: nios_zones.yml
```

Comment:
Split the tests up into targets named after each module being tested. Otherwise the tests will not run when changes to the modules are made.